src/dsp_tools/commands/excel2json/models/input_error.py

```diff
@@ -192,12 +192,52 @@ def execute_error_protocol(self) -> str:
         )
 
 
+@dataclass
+class ResourceClassSheet:
+    """This class contains information if the excel contains two sheets with 'classes' in the name."""
```

Comment:
So, "Englishclasses" or "Frenchclasses" would not be accepted? I think that the only thing which should be forbidden is "classes" itself.

Reply:
Yes, I agree this is what i am going to change now. But this was actually already forbidden previously...